app/assets/stylesheets/layouts/starrydark.scss

```diff
@@ -49,9 +49,9 @@ $ODD_COLOR: #25262E;
   }
 }
 
-.post-navheader .view-button, .view-button.selected, .view-button, .link-box.green, .link-box.blue, .link-box.red, .tag-item { background-color: #463C65; }
+.post-navheader .view-button, .view-button.selected, .view-button, .link-box.action-new, .link-box.action-edit, .link-box.action-favorite, .link-box.action-delete, .link-box.action-dismiss, .tag-item { background-color: #463C65; }
 .flash.breadcrumbs .link-box, .subber .link-box { background-color: #5C5778; }
-#post-editor .green { background-color: #484357; color: $FONT_COLOR; }
+#reply-icon-selector { background-color: #484357; color: $FONT_COLOR; } // replies.scss
```

Comment:
Is the comment here still necessary? (It's not in, say, `starry.scss`)

Reply:
...that's. part of my notes to self for refactoring/stripping starrydark. why is it here -- oh, I think I was noting that for that, okay.